pkgs/development/python-modules/pycdio/default.nix

```diff
@@ -29,23 +32,33 @@ buildPythonPackage rec {
     patchShebangs .
   '';
 
+  build-system = [ setuptools ];
+
   nativeBuildInputs = [
-    nose
-    pkgs.pkg-config
-    pkgs.swig
+    pkg-config
+    swig
   ];
-  buildInputs = [
-    setuptools
-    pkgs.libcdio
-  ] ++ lib.optional stdenv.isDarwin pkgs.libiconv;
+
+  dependencies = [ libcdio ] ++ lib.optional stdenv.isDarwin libiconv;
```

Comment:
Nit: `libiconv` isn’t a Python dependency so it should be in `buildInputs`.

Reply:
Actually maybe it needs to be `propagatedBuildInputs`? I don’t really know how Python extension modules in Nixpkgs work. Hopefully someone who does can comment.